supermicro_scripts/Containerfile

```diff
@@ -0,0 +1,23 @@
+FROM registry.fedoraproject.org/fedora:35
+MAINTAINER Mario Vazquez <mavazque@redhat.com>
+
+RUN echo -ne "[racadm]\nname=Racadm\nbaseurl=http://linux.dell.com/repo/hardware/dsu/os_dependent/RHEL8_64\nenabled=1\ngpgcheck=0" > /etc/yum.repos.d/racadm.repo
```

Comment:
This shouldn't be required for SuperMicro.

Reply:
Sorry, ignore the Containerfile I just copied the whole directory dell_scripts.

The motherboard I've been testing with in my lab is a workstation one, model X11scz-f